app/src/main/java/org/mozilla/reference/browser/library/LibraryItemView.kt

```diff
@@ -22,6 +22,9 @@ import org.mozilla.reference.browser.ext.loadIntoView
  */
 
 interface MultiSelectionInteractor<T> {
+
+    fun open(item: T)
+
```

Comment:
Why did you restore the old method if you can do the same just adding an extra parameter to `openHistoryItems`?

Reply:
I have commented here https://github.com/cliqz/daisy/pull/84/files#r411420623